clusterloader2/testing/load/config.yaml

```diff
@@ -1,12 +1,29 @@
+#Constants
+{{$BIG_GROUP_SIZE := 250}}
+{{$MEDIUM_GROUP_SIZE := 30}}
+{{$SMALL_GROUP_SIZE := 5}}
+{{$PODS_PER_NODE := 30}}
+{{$NODES_PER_NAMESPACE := 100}}
+#Variables
+{{$totalPods := MultiplyInt $PODS_PER_NODE .Nodes}}
```

Comment:
I'm against of that - I would much prefer having something simple, potentially imposing some other conditions on the setup.

So here is my thoughts:

If you assume number of nodes is divisible by 100, then:
bigRCs = num_nodes * 0.03
mediumRCs = num_nodes * 0.3
smallRCs = num_nodes * 3

Now if you assume that NODES_PER_NAMESPACE is a divisor of NUM_NODES, what you get is:
bigRCs = NODES_PER_NAMESPACE * NUM_NAMESPACES * 0.03
which means:
bigRCsPerNamespace = NODES_PER_NAMESPACE * 0.03

So with just two additional requirements:
- NUM_NODES divisable by 100
- NODES_PER_NAMESPACES being a divisor of 100

you can have it much much simple.

And our gole for these tests isn't to have a super generic config that handles all cases and noone understands it.
The purpose is to have it as simple as possible.

And I think that would exactly result in that.

@shyamjvs - thoughts?

Reply:
>bigRCs = num_nodes * 0.03
mediumRCs = num_nodes * 0.3
smallRCs = num_nodes * 3

It looks simpler, but...
- This numbers are created based on assumption on pods per nodes and RC size. If one of this variable is change, updating those equations for number of RC is also required. Based on how this equations look it's not obvious that they need to be updated.
- Having slightly more complex equations allows to show the logic behind it. People will understand this equations instead of guessing why we used those constants.

Reply:
It would be true if we would be changing those numbers. But we aren't and tbh people won't really be doing that.
So introducing complexity just to make it more generic, knowing that probably noone will take advantage of that is overkill.

If needed we can always change that later - fo now, it's overkill that:
- makes it hard to parse - seriously, if I didn't know what you're going to achieve I wouldn't easily parse that
- noone will take advantage of

Reply:
Krzysiek has a point with making it more understandable, but I agree more with @wojtek-t that we should try to keep it as simple as possible - optimizing for generic scenario is probably an overkill for us atm.

Reply:
I'm fine with some comment with a link to a doc where how it's computed and maybe what you currently have if someone would like to use that in the future.

But for now, I would like to keep it as simple as possible.

Reply:
I've changed it to simpler version. However, I've also added comments to these variables.

Reply:
@wojtek-t  Are you ok with this change?